addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/NeatoBindingConstants.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.